src/WormholeL1VotePool.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {L1VotePool} from "src/L1VotePool.sol";
+
+contract WormholeL1VotePool is L1VotePool {
+  /// @param _governor The address of the L1 Governor contract.
+  constructor(address _governor) L1VotePool(_governor) {}
+
+  /// @notice Receives a message from L2 and saves the proposal vote distribution.
+  /// @param payload The payload that was sent to in the delivery request.
+  function _receiveCastVoteWormholeMessages(
```

Comment:
This method was removed from `L1VotePool`, and added here as `internal`. This contract is not abstract, so AFAICT this method is never used?

Reply:
this contract is inherited by WormholeL1ERC20Bridge.sol where the method is used.

@alexkeating and I were discussing folding the WormholeL1VotePool contract (which really only contains this method) into the WormholeL1ERC20Bridge. But because of a couple factors, one of which is a test refactoring, we'll defer on this for now